packages/valory/skills/termination_abci/models.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains the shared state for TerminationAbci."""
+
+from typing import Any
+
+from packages.valory.skills.abstract_round_abci.models import BaseParams
+from packages.valory.skills.abstract_round_abci.models import (
+    BenchmarkTool as BaseBenchmarkTool,
+)
+from packages.valory.skills.abstract_round_abci.models import Requests as BaseRequests
+from packages.valory.skills.abstract_round_abci.models import (
+    SharedState as BaseSharedState,
+)
+from packages.valory.skills.termination_abci.rounds import TerminationAbciApp
+
+
+class SharedState(BaseSharedState):
+    """Keep the current shared state of the skill."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the state."""
+        super().__init__(*args, abci_app_cls=TerminationAbciApp, **kwargs)
+
+
+class TerminationParams(BaseParams):
```

Comment:
Here do `TerminationParams(BaseParams, TransactionParams)` so you don't have to do it here (and therefore in each app that uses these params) https://github.com/valory-xyz/open-autonomy/pull/1459/files#diff-484c7525547a1274d651241dba29755a16df071438a99014a967dfa2aecb6edeR53-R54.